source/_docs/installation/archlinux.markdown

```diff
@@ -5,6 +5,26 @@ description: "Installation of Home Assistant on your Arch Linux computer."
 
 [Arch Linux](https://www.archlinux.org/) is a lightweight and flexible Linux distribution for x86_64.
 
+There are two ways to install Home Assistant on Arch Linux:
+
+## 1. Use existing package
```

Comment:
I'm not so sure that we should add this method. Definitively not as the first option and then only with a fat warning.

There are multiple issues with the Arch Linux package like with every other package that is provided by a distribution. First (which is the primary blocker to include it in Fedora) is the mixing of Python modules. The installer prepares them for the system-wide usage while during run-time Home Assistant is pull the additional packages. Home Assistant has no access to the system's installer, is not running as root and can't use the Python default installation path thus it's very likely that this mixing is causing problems later.

The Arch package is removing the pinning which indicates that Home Assistant and Arch Linux already have different versions. This will make it very hard to debug.

Reply:
I wasn't aware of the issues you mentioned. As i am using the package on my installation and not having any issues so far i thought i would recommend this installation method.
If you think this pull request is wrong you can close it.